app/models/sms_invitation.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class SmsInvitation < Invitation
+  def send_invite(plain_text_token)
+    SendSms.run!(send_sms_attributes(plain_text_token))
+  end
+
+  private
+
+  def send_sms_attributes(plain_text_token)
+    {
+      number: invitation_set.person.mobile_phone,
+      text: generate_message(plain_text_token),
```

Comment:
je zou er nog voor kunnen kiezen om die `generate_message` functie hierheen te halen, aangezien je die volgens mij alleen in de `sms_invitation` class gebruikt

Reply:
Check.

Reply:
Fixed.